sympy/integrals/transforms.py

```diff
@@ -13,6 +13,7 @@
 from sympy.simplify import simplify
 from sympy.utilities import default_sort_key
 from sympy.matrices.matrices import MatrixBase
+from sympy import limit
```

Comment:
Maybe this could be `from sympy.series ...` following the same pattern as above.